lib/Reader/reader.cpp

```diff
@@ -6399,6 +6399,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
 
   while (CurrentOffset < fgNodeGetEndMSILOffset(Fg)) {
     CORINFO_RESOLVED_TOKEN ResolvedToken;
+    bool IsCall = false;
```

Comment:
I think that having this up so high is confusing. I'd just inline the declaration at the sites that use it (at that point, I think the resulting declarations could also be made const).